### ISSUE TITLE
Bitnami Redis Sentinel Auth Support with Harbor 2.5.0

### ISSUE THREAD

**Issue by hikkart** [goharbor/harbor#16907]
Hello All,

Greetings of the day !

Does Harbor 2.5.0 support `Bitnami Redis Sentinel Auth` enabled?

Kindly help me to enable Sentinel Auth! Thanks in advance !

**usecase:**

I am using `Bitnami Redis Sentinel`  helm with Chart Version :`16.8.9` and App Version:`6.2.7`.

With the following configuration in values.yaml, the redis failover (i.e. switch-master) is working fine.

`--set sentinel.enabled=true  --set auth.enabled=true --set auth.sentinel=true` 

But when I am connecting above Redis Sentinel to Harbor Helm with Chart Version: `1.6.1` and App Version: `2.5.0` I am getting following error. 

**values.yaml in harbor helm chart:**
```
redis:
    external:
    # support redis, redis+sentinel
    # addr for redis: <host_redis>:<port_redis>
    # addr for redis+sentinel: <host_sentinel1>:<port_sentinel1>,<host_sentinel2>:<port_sentinel2>,<host_sentinel3>:<port_sentinel3>
    addr: redis-sentinel-node-0.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-1.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-2.redis-sentinel-headless.harbor-space.svc.cluster.local:26379
```

**Error:**
` redigo: no sentinels available; last error: NOAUTH Authentication required.`

```
k8s-user@minikube-ubuntu:~/harbor-setup/git/harbor-setup/Harbor$ k logs kla-harbor-core-7f595f567f-65tq6
2022-05-23T05:28:28Z [INFO] [/core/controllers/base.go:155]: Config path: /etc/core/app.conf
2022-05-23T05:28:29Z [INFO] [/core/main.go:180]: initializing cache ...
2022-05-23T05:28:29Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis-sentinel-node-0.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-1.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-2.redis-sentinel-headless.harbor-space.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 500ms : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-05-23T05:28:29Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis-sentinel-node-0.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-1.redis-sentinel-headless.harbor-space.svc.cluster.local:26379,redis-sentinel-node-2.redis-sentinel-headless.harbor-space.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 718.832613ms : redigo: no sentinels available; last error: NOAUTH Authentication required.

```
>NOTE: If  `Bitnami Redis Sentinel Auth` is disabled , `--set sentinel.enabled=true  --set auth.enabled=false --set auth.sentinel=false` then Harbor is able to connect to Redis Sentinel properly.

**Comment by wy65701436** [comment]
keep monitoring the release, it's in the v2.5.1 and will be released soon.

**Comment by nfoucha** [comment]
This appears to not have been fixed in 2.5.1 as advertised:

`Image:          goharbor/harbor-core:v2.5.1`

```
2022-06-10T22:53:15Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 500ms : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-06-10T22:53:15Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 749.861159ms : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-06-10T22:53:16Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 1.610321123s : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-06-10T22:53:17Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 2.434449142s : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-06-10T22:53:20Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 3.751887509s : redigo: no sentinels available; last error: NOAUTH Authentication required.
2022-06-10T22:53:24Z [ERROR] [/lib/cache/cache.go:110]: failed to ping redis+sentinel://redis:xxxxx@redis-headless.redis.svc.cluster.local:26379/mymaster/0?idle_timeout_seconds=30, retry after 5.809751595s : redigo: no sentinels available; last error: NOAUTH Authentication required.
```

**Comment by wy65701436** [comment]
@nfoucha you have to use the `goharbor/registry-photon:v2.5.1`
